apps/scan/frontend/src/screens/poll_worker_screen.tsx

```diff
@@ -489,8 +434,23 @@ export function PollWorkerScreen({
     );
   }
 
+  // compile-time check for completeness
+  assertFalsy(pollWorkerFlowState);
+
   const commonActions = (
     <React.Fragment>
+      {pollsState !== 'polls_closed_initial' && (
+        <ReprintReportButton
+          printerInfo={printerInfo}
+          scannedBallotCount={scannedBallotCount}
+          lastPollsTransition={lastPollsTransition}
+          reportLabel="Previous"
```

Comment:
Could we put the actual name of the report in this label easily? E.g. in the case you showed in the video I think it would be clearer for this button to say "Print Polls Closed Report"

Reply:
It used to say that, but I removed it just to keep it simple. Yeah, I can totally add it back it.

Reply:
fa6773de1

https://github.com/votingworks/vxsuite/assets/37960853/61caa631-35ee-48a0-9c77-f6e81e65ee6d

Reply:
fa6773de1

https://github.com/votingworks/vxsuite/assets/37960853/61caa631-35ee-48a0-9c77-f6e81e65ee6d